lib/mcb/commands/az/apps/psql.rb

```diff
@@ -1,5 +1,9 @@
 name 'psql'
 summary 'connect to the psql server for an app'
+# h is used for help so used z instead
+option :z, 'host', 'override hostname of database - useful for connecting to restored copies.'\
```

Comment:
Does "-H" work? That'd be preferable if so.

Reply:
good idea, I'll give it a go

Reply:
yep that works, nice one